payments/credit-card/index.md

```diff
@@ -25,31 +25,46 @@ TODO: This page needs serious clean-up.
 {% include alert.html type="warning"
                       icon="warning"
                       header="Site under development"
-                      body="This section of the Developer Portal is under construction and should not be used to integrate against Swedbank Pay's APIs yet." %}
+                      body="This section of the Developer Portal is
+                      under construction and should not be used to integrate
+                      against Swedbank Pay's APIs yet." %}
 
-{% include jumbotron.html body="**Credit Card Payments** is the most popular,
-versatile and global way to initate a transaction with a customer. Choose
-between our **Seamless View**, **Redirect**, or **Direct** integration options." %}
+{% include jumbotron.html body="**Card Payments** is the most popular, versatile
+and global way to initate a transaction with a customer. Choose between our
+**Seamless View**, **Redirect**, or **Direct** integration options." %}
+
+> **Seamless View** is our solution for a payment experience that is integrated
+  directly on your website. The payment process will be executed in an iFrame on
+  your page.
+
```

Comment:
Delete superfluous newline.

Reply:
The newlines are on purpose the separate the three descriptions

Reply:
They should be picked up by your linter.